### ISSUE TITLE
Custom Trigger activation should be a feature in the UI

### ISSUE THREAD

**Issue by akgis32** [WeakAuras/WeakAuras2#5275]
**Is your feature request related to a problem? Please describe.**
Custom Trigger activation should be a UI feature with checkboxes, I too began using WA properly a couple of weeks ago and this made such confusion it was all triggers or any triggers, only found the solution by joining discord and asked there is even a bot comand and since I joined couple days ago I see this being asked all the time.

**Describe the solution you'd like**
Check boxes on individual triggers for trigger activation.

**Describe alternatives you've considered**
The backend already supports it since we can use a custom trigger activation code, Its just a UI thing and I feel this would help so much new ppl starting making their own auras.

**Additional context**
TLDR: https://github.com/WeakAuras/WeakAuras2/wiki/Aura-Activation-and-Deactivation#custom this but in the UI :D
